style.css

```diff
@@ -1,3 +1,98 @@
-h1 {
-  color: aquamarine;
+:root {
+  --purple: #6070ff;
+  --mainFont: 'Poppins';
+}
+
+html {
+  box-sizing: border-box;
+  font-family: var(--mainFont);
+  font-size: 62.5%;
+}
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 85%;
+  margin: 0 auto;
+}
+
+header p,
+nav {
+  font-size: 2rem;
+  color: var(--purple);
+}
+
+header p {
+  font-weight: 700;
+}
+
+/* Hero */
+
+.hero {
+  background-image: url('./img/hero.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 57.5rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.hero h1 {
+  font-size: 4.5rem;
+  font-weight: 700;
+  margin-bottom: 0.5px;
+}
+
+.hero-text p {
+  font-size: 2rem;
```

Comment:
Your font-size is larger than the size provided in Figma. Kindly follow Figma fonts, sizes, etc.